docs/api-reference/effects/glfx-filters.md

```diff
@@ -0,0 +1,343 @@
+# glfx Filters
+
+
+Image effects packaged as reusable shader modules based on the [glfx library](http://evanw.github.io/glfx.js/).
```

Comment:
This page does not mention:

- What npm package this is
- How to use the filters

Reply:
did some changes, there is no official API in luma.gl to use these effects

Reply:
> did some changes, there is no official API in luma.gl to use these effects

I guess the `_ShaderModulePass` is still experimental. Still, it is strange to list these with no information on how to use them. Maybe add a link to deck.gl effects page?

Reply:
yeah, that's better, will add a link here when the deck.gl effects doc done.